app/controllers/application_controller.rb

```diff
@@ -1,2 +1,14 @@
 class ApplicationController < ActionController::Base
+	before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
2 spaces for indentation are enough here and in the rest of the file

Reply:
<img width="708" alt="image" src="https://user-images.githubusercontent.com/108671294/182141676-16523abf-d401-4673-8c2c-533b64aa15c0.png">